CSharp/DevVmPowershell/DevVmPsModules/DocumentModule.cs

```diff
@@ -114,14 +90,14 @@ private void ValidateInputArguments()
 				throw new ArgumentException(nameof(WorkspaceId), $"{nameof(WorkspaceId)} cannot be less than or equal to 0.");
 			}
 
-			if (string.IsNullOrWhiteSpace(FileType) && (FileType.ToLower().Contains("documents") || FileType.ToLower().Contains("images")))
+			if (string.IsNullOrWhiteSpace(FileType))
 			{
 				throw new ArgumentNullException(nameof(FileType), $"{nameof(FileType)} cannot be NULL or Empty.");
 			}
 
-			if (!FileType.ToLower().Contains("documents") && !FileType.ToLower().Contains("images"))
+			if (!FileType.ToLower().Equals("document", StringComparison.OrdinalIgnoreCase) && !FileType.ToLower().Equals("image", StringComparison.OrdinalIgnoreCase))

```

Comment:
can we add document and image to the constant class? Also resuse the same constants for TestCases